Files/Views/SettingsPages/Preferences.xaml

```diff
@@ -137,5 +130,7 @@
                 </StackPanel>
             </StackPanel>
         </ScrollViewer>
+
+        <dialogs:RestartDialog x:Name="RestartDialog"/>
```

Comment:
Can you add a light dismiss to the notification?

Reply:
@yaichenbaum I'm gonna file an issue in the WindowsCommunityToolkit repo to fix dark dismiss button on dark theme.

Reply:
I meant light dismiss as in dismissing the notification if you click outside of the notification.

Reply:
Notification closes after 10 seconds